### PR TITLE
drivers: change tree trace level

### PR DESCRIPTION
**Pull request by ppaillet**
Change the clock and regulator trace level of the print tree functions so that they can be seen when requested by xtest without needing to activate all debug traces.

### CODE REVIEW

**Comment by GseoC** [comment]
Hi Pascal, could specify in commit messages which xtest you're referring to (the one linked with PTA stats).
Otherwise: `Acked-by: Gatien Chevallier <gatien.chevallier@foss.st.com>`

File: core/drivers/clk/clk.c [hunk 1]
```diff
@@ -540,9 +540,8 @@ static void print_tree(void)
 
 void clk_print_tree(void)
 {
-	if (IS_ENABLED(CFG_DRIVERS_CLK_PRINT_TREE) &&
-	    TRACE_LEVEL >= TRACE_DEBUG) {
-		DMSG("Clock tree summary (informative):");
+	if (IS_ENABLED(CFG_DRIVERS_CLK_PRINT_TREE)) {
```
Comment (etienne-lms) [inline core/drivers/clk/clk.c:543]: Removing test on `TRACE_LEVEL >= TRACE_INFO` makes that when `CFG_DRIVERS_CLK_PRINT_TREE` is enabled but `TRACE_LEVEL` to low, there will be useless code embedded.
I would be in favor of restoring this test.

Ditto in `core/drivers/regulator/regulator.c`.

**Review by etienne-lms — COMMENTED** [review]
I think it is a legitimate change. Enabling debug trace level may generate quite many trace messages.

Could you also update the header files:
`clk.h`:
```patch
 /**
  * clk_print_tree() - Print current clock tree summary to output console
  *
-  * The clock is printed with the debug trace level.
+  * The clock is printed with the info trace level.
  */
```

`regulator.h`:
```patch
-/* Print current regulator tree summary to console with debug trace level */
+/* Print current regulator tree summary to console with info trace level */
```
and `mk/config.mk`:
```patch
 # CFG_DRIVERS_CLK_PRINT_TREE embeds a helper function to print the clock tree
-# state on OP-TEE core console with the debug trace level.
+# state on OP-TEE core console with the info trace level.
 (...)
 # CFG_DRIVERS_REGULATOR_PRINT_TREE embeds a helper function to print the
 # regulator tree state on OP-TEE core console with the info trace level.
```
(note that info level was already mentioned for regulator tree traces :)

File: core/drivers/clk/clk.c (hunk 1 above)
Comment (ppaillet) [reply core/drivers/clk/clk.c]: done, thank you!

**Comment by ppaillet** [comment]
> Hi Pascal, could specify in commit messages which xtest you're referring to (the one linked with PTA stats). Otherwise: `Acked-by: Gatien Chevallier <gatien.chevallier@foss.st.com>`

done, thank you!

**Review by etienne-lms — COMMENTED** [review]
`Reviewed-by: Etienne Carriere <etienne.carriere@foss.st.com>`

**Comment by etienne-lms** [comment]
Hi @jforissier, could you enable CI tests for this P-R?
Are you fine with the changes?

**Comment by jforissier** [comment]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by etienne-lms** [comment]
Thanks @jforissier for your feedback.
Could you enable CI tests for this P-R? It should be ready for merging, let's check CI tests are still happy.

**Pull request merged**